scrapli_community/cisco/asa/cisco_asa.py

```diff
@@ -0,0 +1,60 @@
+"""scrapli_community.cisco.asa.cisco_asa"""
+
+from scrapli.driver.network.base_driver import PrivilegeLevel
+from scrapli_community.cisco.asa.async_driver import default_async_on_close, default_async_on_open
+from scrapli_community.cisco.asa.sync_driver import default_sync_on_close, default_sync_on_open
+
+DEFAULT_PRIVILEGE_LEVELS = {
+    "exec": (
+        PrivilegeLevel(
+            pattern=r"^[\S]{1,63}>(\s|)$",
+            name="exec",
+            previous_priv="",
+            deescalate="",
+            escalate="",
+            escalate_auth=False,
+            escalate_prompt="",
+        )
+    ),
+    "privilege_exec": (
+        PrivilegeLevel(
+            pattern=r"^[\S]{1,63}#(\s|)$",
```

Comment:
Can this be more explicit? I see there is `\S` here and exec but some specific char classes for config -- can we use the pattern from config? `\S` is easier to read for sure, but ive been trying to keep these as explicit as possible even if it makes the readability a bit shit. It just feels "safer" that way... perhaps all in my head 🤣

Reply:
Definitely agree, have fixed this as well.